src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int numberOfBalls = 3;
```

Comment:
```suggestion
    private static final int NUMBER_OF_BALLS = 3;
```

constant should has all upper-case letter